### PR TITLE
[Brl.FileSystem] Add filetime getter/setter utilizing SDateTime

### PR DESCRIPTION
**Pull request by GWRon**

### CODE REVIEW

**Comment by GWRon** [comment]
I created an overload for `SetFileTime()` but also a `SetFileDateTime()` to counterpart to the additionally added `GetFileDateTime()`.

**Comment by GWRon** [comment]
I am not sure if performancewise it would be smarter for the "setter" to use `dateTime:SDateTime var` so only a reference is used, not a struct copy.

File: filesystem.mod/filesystem.bmx [hunk 1]
```diff
@@ -284,6 +284,73 @@ Function SetFileTime( path:String, time:Long, timeType:Int=FILETIME_MODIFIED)
 	End If
 End Function
 
+Rem
+bbdoc: Sets the file modified or last accessed time.
+about: @dateTime is the basic DateTime struct defined in pub.stdc .
+End Rem
+Function SetFileTime( path:String, dateTime:SDateTime, timeType:Int=FILETIME_MODIFIED)
```
Comment (woollybah) [inline filesystem.mod/filesystem.bmx:291]: There's no need to duplicate all the code each time. You can just call something like `SetFileTime( path, dateTime.ToEposhSecs(), timeType)`

**Comment by GWRon** [comment]
Yes. But this adds a function call (overhead). 
But yeah, thought of keeping it simple too :-)

File: filesystem.mod/filesystem.bmx (hunk 1 above)
Comment (GWRon) [reply filesystem.mod/filesystem.bmx:291]: Incorporated your suggestion.

**Pull request merged**